DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/AllDelegates/AllDelegateItemsViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.AllDelegates
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
+
+    public class AllDelegateItemsViewModel : BaseJavaScriptFilterableViewModel
+    {
+        public readonly IEnumerable<SearchableDelegateViewModel> Delegates;
+
+        public AllDelegateItemsViewModel(
+            int centreId,
+            IEnumerable<DelegateUserCard> delegateUserCards,
+            IEnumerable<(int id, string name)> jobGroups,
+            CentreCustomPromptHelper customPromptHelper
+        )
+        {
+            Delegates = delegateUserCards.Select(
+                delegateUser =>
+                {
+                    var customFields = customPromptHelper.GetCustomFieldViewModelsForCentre(centreId, delegateUser);
+                    var closedCustomPrompts = customPromptHelper.GetClosedCustomPromptsForCentre(centreId);
+                    return new SearchableDelegateViewModel(delegateUser, customFields, closedCustomPrompts);
+                }
+            );
+
+            var customPrompts = customPromptHelper.GetClosedCustomPromptsForCentre(centreId).ToList();

```

Comment:
I think this would be the same as closedCustomPrompts above, and so could be removed here if it is passed in.